usr/share/rear/finalize/Linux-ppc64le/680_install_PPC_bootlist.sh

```diff
@@ -47,7 +47,7 @@ fi
 
 if [[ ${#boot_list[@]} -gt 0 ]]; then
     LogPrint "Set LPAR bootlist to '${boot_list[@]}'"
```

Comment:
Using `${boot_list[@]}` inside double quotes will cause it to expand to multiple strings, like this:
```
boot_list=( foo bar )
"Set LPAR bootlist to '${boot_list[@]}'" -> "Set LPAR bootlist to 'foo"   "bar'"
```
LogPrint actually handles multiple arguments properly, so it is not a bug. But it is IMO better to do this in a less surprising way, because I don't think this kind of splitting into multiple argument was intended. And it can become a bug when this code gets cargo-culted to other places. And it is flagged by static analysis: https://github.com/koalaman/shellcheck/wiki/SC2145.
I propose:
```
LogPrint "Set LPAR bootlist to '${boot_list[*]}'"
```